src/Tickets/Commerce/Hooks.php

```diff
@@ -83,7 +83,7 @@ protected function add_actions() {
 
 		add_action( 'tec_tickets_commerce_order_status_transition', [ $this, 'modify_tickets_counters_by_status', ], 15, 3 );
 
-		add_action( 'admin_bar_menu', [ $this, 'include_admin_bar_test_mode' ], 1000, 1 );
+		add_action( 'admin_bar_menu', [ $this, 'include_admin_bar_test_mode' ], 5, 1 );
```

Comment:
I wonder if we should add a conditional here for before 6.6 to stay as is and after 6.6 to use priority `5`.

Also add changelog and point to next release.